### PR TITLE
tiny typo fix

### PR DESCRIPTION
**Pull request by postylem**
`fomat` -> `format` in a couple places :)

### CODE REVIEW

File: diagram-generator/README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ cxy, f = axs[1].cohere(s1, s2, 256, 1. / dt)
 axs[1].set_ylabel('coherence')
 
 fig.tight_layout()
-plt.savefig("$DESTINATION$", dpi=300, fomat="$FORMAT$")
+plt.savefig("$DESTINATION$", dpi=300, fotmat="$FORMAT$")
```
Comment (alerque) [inline diagram-generator/README.md:205]: `Fotmat` awhata?

**Review by alerque — CHANGES_REQUESTED** [review]
Thanks for the contributions, but lets actually fix typos with real words instead of more typos ;-)

File: diagram-generator/README.md (hunk 1 above)
Comment (postylem) [reply diagram-generator/README.md:205]: oh my word

**Review by postylem — COMMENTED** [review]
Yikes. I fixed it now.

**Comment by postylem** [comment]
Okay I fixed that typo.  Yikes. Who knew 'format' was so hard to spell?

**Review by alerque — APPROVED** [review]

**Pull request merged**